### ISSUE TITLE
Does UIRecord provide screenshots as soon as an item is clicked?

### ISSUE THREAD

**Issue by xbedna72** [microsoft/WinAppDriver#1314]
Hi,
I am so sorry if this question has already been asked. I am just curios. Does WinAppDriverUIRecorder create screenshot immediately after click on an item? Or does exist some modul which provides this function? I am looking forward to your answers or ideas.

Greetings
Maya

**Comment by naeemakram** [comment]
No, the UI Recorder doesn't provide screenshot. You can use Appium Desktop Client in conjunction with WinAppDriver to capture screenshots. You can learn more about it [here]([url](https://youtu.be/GglD59kq2pQ))(YouTube) and [here](http://aprogrammersday.blogspot.com/2020/05/appium-client-for-windows-automation.html).
